qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddEditNewPrivilegePage.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class AddEditNewPrivilegePage extends Page{
+	protected static final String PAGE_URL = "/adminui/metadata/privileges/privilege.page?action=add&";
+	public static final By ENTER_PRIVILEGE_FIELD = By.cssSelector("#privilege-field");
+	public static final By ENTER_DESCRIPTION_FIELD = By.cssSelector("#description-field");
```

Comment:
@mherman22 take a close look at the variable names

```js
public static final By ENTER_PRIVILEGE_FIELD = By.cssSelector("#privilege-field");
public static final By ENTER_DESCRIPTION_FIELD = By.cssSelector("#description-field");
```

Reply:
something like **ENTER_PRIVILEGE_NAME_FIELD**?

Reply:
what do you think of `ENTER_PRIVILEGE_FIELD` vs `PRIVILEGE_FIELD`?